src/sketchup-stl/exporter.rb

```diff
@@ -51,7 +51,18 @@ def self.select_export_file
         filename
       end
 
-      def self.export(path, options = OPTIONS)
+      def self.export(path = nil, options = OPTIONS)
+        if options['selection_only']
+          export_ents = Sketchup.active_model.selection
+        else
+          export_ents = Sketchup.active_model.active_entities
+        end
+        if export_ents.length == 0
+           UI.messagebox("There is nothing to export. Either the model or the selection is empty.")
+           return
+        end
+        path = select_export_file() if (path.nil? or path.empty?)
```

Comment:
The brackets is probably not needed if you use `||` instead of `or`. Also, the style guide dictates to use `||` and `&&` over `or` and `and`: https://github.com/styleguide/ruby